requirements.txt

```diff
@@ -29,6 +29,11 @@ six>=1.15.0
 # -------------------------------------------------------------------
 # pyserial-asyncio==0.4.0;python_version>="3.4"
 
+# -------------------------------------------------------------------
+# if you want to use the Trio asynchronous version, uncomment these
+# -------------------------------------------------------------------
+# async_generator ~= 1.10
```

Comment:
can we add this as part of `extra_requires` in [setup.py](https://github.com/riptideio/pymodbus/blob/master/setup.py#L76)

Reply:
Certainly, that's how I usually do it but managed to overlook that here.  `:[`  Thanks for catching it.